src/packet/create.ts

```diff
@@ -0,0 +1,58 @@
+import { randomBytes } from "bcrypto/lib/random";
+import sha256 = require("bcrypto/lib/sha256");
```

Comment:
require => import

Reply:
I want to experiment with just using typescript for transpilation in this repo, instead of using babel.